src/meanshift.jl

```diff
@@ -0,0 +1,153 @@
+
+type MeanShiftResult{T <: FloatingPoint} <: ClusteringResult
```

Comment:
`FloatingPoint` is soon to be deprecated in favor of `AbstractFloat`. See https://github.com/JuliaLang/julia/pull/12162